packages/webpack-cli/package.json

```diff
@@ -29,7 +29,6 @@
   "dependencies": {
     "@discoveryjs/json-ext": "^0.5.0",
     "@webpack-cli/configtest": "^1.0.0",
-    "@webpack-cli/info": "^1.2.1",
```

Comment:
No need to touch it

Reply:
Oops, thought it to be `init`. reverting.